handlers/bootstrap.go

```diff
@@ -32,6 +31,15 @@ func ActivateMachine(event *events.Event, apiClient *client.RancherClient) (err
 		"eventId":    event.Id,
 	}).Info("Activating Machine")
 
+	if err := acquire_networked_global_persistent_lock(event.ResourceId); err != nil {
+		log.WithFields(log.Fields{
+			"err": err,
+		}).Info("Could not acquire global lock")
+		reply := newReply(event)
+		return publishReply(reply, apiClient)
```

Comment:
@wlan0 Would replying to this event cause cattle to think the process completed successfully?
...If that's the case, we probably dont want to reply with an error either.
Maybe the right thing to do is to drop the event without sending any reply.
May need @ibuildthecloud to weigh in on this.